examples/hosts/app-integration/external-data/src/model/containerCode.ts

```diff
@@ -27,18 +25,20 @@ const SignalType = {
  * {@inheritDoc ModelContainerRuntimeFactory}
  */
 export class TaskListContainerRuntimeFactory extends ModelContainerRuntimeFactory<IAppModel> {
-	public constructor() {
+	private readonly _taskListId: string;
+	public constructor(taskListId: string) {
```

Comment:
```suggestion
	public constructor(
	  private readonly taskListId: string) {
```
We can declare a property "inline" in the constructor parameters with all its necessary access modifiers and avoid the extra assignment in line 33.